.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # Primary owner should be listed first in list of global owners, followed by any secondary owners
-*       @SirSpidey @ocofaigh
+*       @chrisw-ibm, @hiltol
```

Comment:
remove the comma as its invalid syntax

Reply:
<img width="268" alt="image" src="https://github.com/user-attachments/assets/c5061fd5-f3cf-45c0-85bc-0602b6a0a967">